pkg/api/validation/validation.go

```diff
@@ -3409,6 +3409,14 @@ func validateContainerResourceName(value string, fldPath *field.Path) field.Erro
 			return append(allErrs, field.Invalid(fldPath, value, "must be a standard resource for containers"))
 		}
 	}
+	if len(strings.Split(value, "/")) == 2 {
+		if !helper.IsNonstandardContainerResourceName(value) && !helper.IsOpaqueIntResourceName(api.ResourceName(value)) {
+			return append(allErrs, field.Invalid(fldPath, value, "must be a nonstandard resource for containers"))
+		}
+	}
+	if len(strings.Split(value, "/")) > 2 {
```

Comment:
I think there is no need to call `strings.Split` and `len` again and again.

Also this part can be treated as an `else` clause.